pandas/tests/frame/indexing/test_where.py

```diff
@@ -1049,3 +1049,12 @@ def test_where_mask_deprecated(frame_or_series):
 
     with tm.assert_produces_warning(FutureWarning):
         obj.mask(mask, -1, errors="raise")
+
+def test_consitency_inplace():
```

Comment:
we could maybe put this test adjacent to `test_setitem_scalar_with_mask_validation` in `pandas/tests/arrays/string_/test_string.py`

Although, a test in this module would be useful if it was parametrized to ensure consistent behavior with other EA array types (would need to check what we already have)

Reply:
@simonjayhawkins Hi, I wanted to mention you and say that I have changed the place of the code as you said. Is adding a parametrized also needed still?